app/views/dashboard.pug

```diff
@@ -2,20 +2,21 @@ doctype html
 html(lang="en")
     head
         title User Dashboard
-        script (src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js")
         link(rel="stylesheet", href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css")
         link(rel="stylesheet", href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css")
+        link(rel="stylesheet", src ="/externalCSS.css")
         script(src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js")
+        script(type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js")
 
-    body(style='width:100%;height:100%;background-image:url("/assets/background.png");')
+    body(id='full' style='background-image:url("/assets/background.png");')
 
-        div.container(style='width:100%;height:100%;')
+        div.container(id='full')
 
-            div(id='topHalf' style='float:top;height:20%;')
+            div(id='topFloat')
                 div(id='mainTitle' style='text-align:center;')
                     h1 Welcome to Hive!
 
-            div(id='bottomHalf' style='position:absolute;width:100%;height:80%;left:0%;top:20%;')
+            div(id='bottomFloat2' style='position:absolute;width:100%;height:80%;left:0%;top:20%;')

```

Comment:
nope that is just classic lazy naming conventions, I didn't know if there was a bottomFloat1 so I made 2 for testing and forgot to fix it. I will fix that.